packaging/installer/methods/source.md

```diff
@@ -0,0 +1,289 @@
+<!--
+---
+title: "Manually build Netdata from source"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/installer/methods/source.md
+---
+-->
+
+# Manually build Netdata from source
+
+These instructions are for advanced users and distribution package
+maintainers. You almost certainly want to [install Netdata from a Git
```

Comment:
To make it clearer, "for anyone else, we have a [building from source guide]()."

Reply:
I'm hesitant to use this exact wording as the user is not actually building from source in that case, and it may cause some confusion with this specific guide.

Perhaps: 'Unless this describes you, you almost certainly want to follow [our guide for manually installing Netdata from a git checkout](/packaging/installer/methods/manual.md) instead.'